keentools_facebuilder/blender_independent_packages/pykeentools_loader/install.py

```diff
@@ -20,18 +20,29 @@
 import os
 import sys
 from threading import Thread, Lock
+from enum import Enum
+import bpy
 
 from .config import *
 
 
-__all__ = ['is_installed', 'uninstall', 'installation_status',
+__all__ = ['PartInstallation', 'is_installed', 'uninstall', 'installation_status',
            'install_from_download', 'install_from_download_async',
-           'install_from_file', 'loaded', 'module']
+           'install_core_from_file', 'install_addon_from_file',
+           'download_file_name', 'download_zip', 'download_zip_async',
+           'updates_downloaded', 'loaded', 'module', 'remove_download',
+           'install_download', 'download_file_version_path',
+           'downloaded_keentools_version']
 
 
 _unpack_mutex = Lock()
 
 
+class PartInstallation(Enum):
```

Comment:
I'm not sure this is a good enum and interface (using it) is good. It can be used inside, but in module interface I don't think this is a good idea